testsuite/features/step_definitions/common_steps.rb

```diff
@@ -287,38 +288,37 @@
   raise 'distro ' + distro + ' does not exist' unless ct.distro_exists(distro)
 end
 
-Then(/^create profile "([^"]*)" as user "([^"]*)" with password "([^"]*)"$/) do |arg1, arg2, arg3|
+Then(/^create profile "([^"]*)" as user "([^"]*)" with password "([^"]*)"$/) do |profile, user, password|
   ct = CobblerTest.new
-  ct.login(arg2, arg3)
-  raise 'profile ' + arg1 + ' already exists' if ct.profile_exists(arg1)
+  ct.login(user, password)
+  raise 'profile ' + profile + ' already exists' if ct.profile_exists(profile)
```

Comment:
I would be caution of changing code lines not related to that API handler.
As this could conflict with other PRs.

Reply:
Yes, it's only style changes, i should have done them in a different commit, and even in a different PR.

I can do that if you want, but it will take time. I wanted to be fast, this is why you have a bit of piggybacking here.

Reply:
As you can see it will collide in  @nodeg PR
https://github.com/uyuni-project/uyuni/pull/5142/files#diff-75af4fb4d32d0e2675676f25807299a41f0afdccc6184ab7c255f4a43852e605R292-R297

I would better revert these lines.

Reply:
ok.